chia/_tests/blockchain/test_blockchain.py

```diff
@@ -61,6 +60,7 @@
     DEFAULT_HIDDEN_PUZZLE_HASH,
     calculate_synthetic_secret_key,
 )
+from chia.wallet.wallet_spend_bundle import WalletSpendBundle
```

Comment:
as far as I can tell, `test_blockchain` is clearly part of the full node. It's not obvious to me why it should have a dependency on the wallet. It doesn't look right.